pkg/apis/build/v1beta1/build_types.go

```diff
@@ -172,6 +172,32 @@ func (buildSpec *BuildSpec) StrategyName() string {
 	return buildSpec.Strategy.Name
 }
 
+// VulnerabilityIgnoreOptions refers to ignore options for vulnerability scan
+type VulnerabilityIgnoreOptions struct {
+
+	// Issues references the security issues to be ignored in vulnerability scan
+	Issues []string `json:"issues,omitempty"`
+
+	// Severity indicates the severities of security issues to be ignored (comma separated)
+	Severity string `json:"severity,omitempty"`
```

Comment:
The idea was a different one. But good that we clarify it. Basically the idea was that you specify only one severity optionally. And then this one and all lower severities will get ignored.

Is it actually possible to specify an enum here so that the schema validation will catch when some unexpected severity is specified?

```suggestion
	// Severity indicates the severity of the issues that should be ignored. Lower severity issues will also be ignored.
	//
	// +optional
	Severity *string `json:"severity,omitempty"`
```